pkg/daemon/ceph/client/mirror.go

```diff
@@ -376,6 +391,14 @@ func EnableRBDRadosNamespaceMirroring(context *clusterd.Context, clusterInfo *Cl
 		return errors.Errorf("ceph version %q does not support mirroring in rados namespace %q with --remote-namespace flag, supported version are v20 and above.", clusterInfo.CephVersion.String(), poolAndRadosNamespaceName)
 	}
 
+	implicitNamespaceKey := "<implicit>"
```

Comment:
Define this in a const so it can be referenced everywhere, instead of duplicating it in multiple places.